src/matter/ControllerNode.ts

```diff
@@ -781,7 +780,10 @@ class Controller implements GeneralNode {
 
     async completeCommissioningForNode(nodeId: NodeId, discoveryData?: DiscoveryData): Promise<AddDeviceResult> {
         if (!this.#commissioningController) {
-            throw new Error(`Can not register NodeId ${nodeId} because controller not initialized.`);
+            return {
+                result: false,
+                error: new Error(`Can not register NodeId ${nodeId} because controller not initialized.`),
```

Comment:
maybe put nodeId in `"` :D

Reply:
I do this when I do full controller cleanup